### ISSUE TITLE
"Point Buy" attribute score generation

### ISSUE THREAD

**Issue by vaxinate** [vaxinate/dndchar#2]
users should be able to choose to min/max ability scores by taking advantage of the "point buy" system
## Optional: Customizing Ability Scores

The method described here allows you to build a character with a set of ability scores you choose individually. Start with an 8 in all six ability scores, then spend 30 points to improve them. The cost of raising a score from 8 to a higher number is shown below.
### Score Cost from 8

9      1
10    2
11    3
12    4
13    5
14    7
15    9
16    12

**Comment by wedgex** [comment]
How do we want this to work in, is there like a point buy mode? I feel like the UI for point buy and rolling would change up a bit.

**Comment by vaxinate** [comment]
Yeah I don't know how this fits in exactly. I'm thinking it works kind of
like the roll button. You hit the "points" button and all your scores get
set to 8 and there's a points indicator. Each score has buttons to bump it
up or down. You can go over 30 pts if you want but it will show you 45/30
or whatever. We could just always calculate the point score for a build
that might be tyte. It might give rolled scores clearer value.
